styles.css

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+/* Nav Bar */
+.toolbar {
+  height: 48px;
+}
+
+.navbar-items {
+  display: flex;
+  height: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 30px;
+  font-family: 'Inter', sans-serif;
+}
+
+.menu-button {
+  margin-right: 30px;
+}
+
+/* Headline */
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  text-align: center;
+  background-image: url("images/header-illsutration-mobile@2x.jpg");
+  background-size: cover;
+  width: 100%;
+  height: 100vh;
+}
+
+.headline h1 {
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  margin: 0 24px;
+}
+
+.headline span {
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  color: #36b37e;
+  margin: 0 24px;
+}
```

Comment:
- [ ] Kindly remove the `margin: 0 24px;` role, to match the figma design.

![image](https://user-images.githubusercontent.com/35535130/158706130-121f1b75-641c-4ebe-97a5-ac1a2f28e69e.png)